cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/system/SystemWorkerHttpHandlerInternal.java

```diff
@@ -101,33 +85,33 @@ public void run(FullHttpRequest request, HttpResponder responder) {
       return;
     }
 
-
     long startTime = System.currentTimeMillis();
-    String className = null;
+    RunnableTaskRequest runnableTaskRequest = GSON.fromJson(request.content().toString(StandardCharsets.UTF_8),
```

Comment:
Is it okay to deserialise outside of try/catch? In case of an exception, we'll not be decreasing the requestProcessedCounter.